src/sim65.vcxproj

```
@@ -86,15 +86,17 @@
     <ClInclude Include="sim65\error.h" />
     <ClInclude Include="sim65\memory.h" />
     <ClInclude Include="sim65\paravirt.h" />
+    <ClInclude Include="sim65\peripherals.h" />
   </ItemGroup>
   <ItemGroup>
     <ClCompile Include="sim65\6502.c" />
     <ClCompile Include="sim65\error.c" />
     <ClCompile Include="sim65\main.c" />
     <ClCompile Include="sim65\memory.c" />
     <ClCompile Include="sim65\paravirt.c" />
+    <ClInclude Include="sim65\peripherals.c" />
```

Perhaps `ClCompile` instead of `ClInclude`?

```suggestion
    <ClCompile Include="sim65\peripherals.c" />
```

👋  @sidneycadot you appeared in my GitHub news feed, couldn't help but poke around :)

---

If this works I owe you a beer :-)

---

All hail the power of a fresh pair of eyes! 🍺🍺🍺